lib/views/tree-view.js

```diff
@@ -244,43 +245,11 @@ class TreeView extends ScrollView {
     this.info.hide();
   }
 
-  loadServers(password) {
-    const self = this;
-
-    // Add detail information in error notification
-    // Uncaught SyntaxError: Unexpected token  in JSON at position 0 #44
-    // https://github.com/h3imdall/ftp-remote-edit/issues/44
-    let configHash = atom.config.get('ftp-remote-edit.config');
-    if (configHash) {
-      let config = decrypt(password, configHash);
-      let sortServerProfilesByName = atom.config.get('ftp-remote-edit.tree.sortServerProfilesByName');
-
-      try {
-        self.servers = JSON.parse(cleanJsonString(config));
-        self.servers.sort((a, b) => {
-          if (sortServerProfilesByName) {
-            if (a.name.toLowerCase() < b.name.toLowerCase()) return -1;
-            if (a.name.toLowerCase() > b.name.toLowerCase()) return 1;
-          } else {
-            if (a.host < b.host) return -1;
-            if (a.host > b.host) return 1;
-          }
-          return 0;
-        });
-        return true;
-
-      } catch (e) {
-        throwErrorIssue44(e, password);
-        return false;
-      }
-    }
-    return true;
-  }
 
   reload() {
     const self = this;
 
-    if (!self.servers) {
+    if (!Storage.getServers()) {
```

Comment:
getServer() always returns a array. The check must therefore be changed: 

if (Storage.getServers().length === 0) {